static/css/todo.css

```diff
@@ -3,62 +3,76 @@
 @import "vars.css";
 
 .todo-container {
-    padding: 10px;
-    border: 2px solid var(--button-background-lm);
-    border-radius: 8px;
+    padding: 0.625rem;
+    border: 0.125rem solid var(--button-background-lm);
+    border-radius: 0.5rem;
     background-color: var(--background-secondary-lm);
     font-family: var(--font-primary);
-    font-size: medium;
+    font-size: 0.875rem;
     color: var(--text-primary-lm);
     display: flex;
-    flex-wrap: wrap;
-    gap: 20px; /* Spacing between columns */
-    margin: 20px auto;
-    overflow: hidden; /* Prevents content from overflowing */
-    max-width: 100%; /* Ensures container doesn't exceed parent width */
+    gap: 1.25rem; /* Spacing between columns */
+    margin: 0.938rem; 
+    overflow-x: auto;
     box-sizing: border-box;
+
+    max-width: 100%; /*Ensures container doesn't exceed parent width*/
+    max-height: 16.25rem;
     align-items: stretch; /* Ensures all columns have the same height */
 }
 
 /* Individual columns */
 .to-do-column {
     flex: 1; /* All columns get equal width */
-    max-width: 300px; /* Restrict maximum width for each column */
-    border: 1px solid #ccc;
-    border-radius: 8px;
-    padding: 10px;
+    max-width: 18.75rem; /* Restrict maximum width for each column */
+    border: 0.063rem solid #ccc;
+    border-radius: 0.5rem;
+    padding: 0.625rem;
     background-color: var(--background-primary-lm);
     display: flex;
     flex-direction: column;
-    gap: 10px;
-    overflow: hidden;
+    gap: 0.625rem;
+    overflow-y: hidden;
     box-sizing: border-box;
-    max-height: 100%; /* Ensure equal height */
+    max-height: 16.875rem; /* Ensure equal height */
```

Comment:
same question as above, do we want the max height to be bigger? These boxes feel a little small to me but up to you!